pkg/crc/services/dns/dns.go

```diff
@@ -40,6 +41,26 @@ func RunPostStart(serviceConfig services.ServicePostStartConfig) (services.Servi
 	// Remove the dnsmasq container if it exists during the VM stop cycle
 	_, _ = serviceConfig.SSHRunner.Run("sudo podman rm -f dnsmasq")
 
+	// Add a virtual dummy network interface and set the IP as `192.168.126.11` which is the IP used to create the bundle
+	if _, err := serviceConfig.SSHRunner.Run("sudo ip link add eth10 type dummy && sudo ip addr add 192.168.126.11/24 dev eth10"); err != nil {
+		result.Success = false
+		result.Error = err.Error()
+		return *result, err
+	}
+	// Todo: Move those to snc script when start building for 4.4 bundles
+	// Add node-ip field in kublet service and set it to the static IP `192.168.126.11`
```

Comment:
So this PR will be changed before we release for 4.4 ?

Reply:
why change? this would not effect 4.3, ...

Reply:
I'm referring to the comment which says some of the changes in this PR really belongs to snc. So I'm asking whether this is going to be changed before this PR is merged or not.

Reply:
right. but if this is part of SNC i would fail to see how this sruvives a 'reboot' without creating a config file or custom systemd unit.